lib/src/migrators/division.dart

```diff
@@ -90,13 +80,7 @@ class _DivisionMigrationVisitor extends MigrationVisitor {
       var expression = node.expression;
       if (expression is BinaryOperationExpression &&
           expression.operator == BinaryOperator.dividedBy) {
-        if (_shouldMigrate(expression)) {
-          addPatch(patchBefore(node, "divide"));
-          _patchParensIfAny(expression.left);
-          _patchSlashToComma(expression);
-          _patchParensIfAny(expression.right);
-        }
-        super.visitBinaryOperationExpression(expression);
+        _visitSlashOperation(expression, surroundingParens: node);
```

Comment:
Rather than passing in `surroundingParens`, which requires `_visitSlashOperation()` to understand a lot about what context it might be called in, consider having this code delete the parentheses itself and just call `_visitSlashOperation(expression)`.